ruby/alias_manager.rb

```diff
@@ -0,0 +1,62 @@
+# ask user for their full name
+# split the string
+# swap between first and last name
+# create a loop that cheks if letter is vowel or consonant
+# 	-if vowel - create a method that finds the next vowel and call it
+# 	-else use .next to replace the consonant.
+continue = true
+new_names = []
+original_names = []
+counter = 0
+vowels = ['a', 'e', 'i', 'o', 'u']
+consonants = ['b', 'c', 'd', 'f', 'g', 'h', 'j', 'k', 'l', 'm', 'n', 'p', 'q', 'r', 's', 't', 'v', 'w', 'y', 'z']
+
+def get_next(arr, letter)
+	result = if arr.include?(letter)
+		new_index = arr.index(letter)
+		if new_index == arr.length - 1
+			arr[0]
```

Comment:
depending on your version of ruby `arr.first` should work here as well.  If it does not then this syntax is fine.